examples/trivial_discord_bot.py

```diff
@@ -0,0 +1,148 @@
+# Remember when wiring random services together to see what you could make happen?
```

Comment:
Overall Q for trivial_discord_bot.py; is this the eventual workflow we're going for with mewbot, or is this a temporary example for dev?

Reply:
My aim when putting it together was very tempoary example for dev. Just wanted something with a working event loop to play around with/preform basic validation of class drafts.

"example" is a bad name for this concept. In future, will tag such things as "scratch_{name}.py"